### DISCUSSION TITLE
How to remove the name of the site from appearing in the title of the article

### DISCUSSION THREAD

**Discussion by abdelkareemkobo** [arthelokyo/astrowind#514] [Q&A]
![image](https://github.com/user-attachments/assets/5c54f648-4969-442c-bae2-79bda2e30b68)
The articles i have inlcude the following : 
name_of_the_article - name of th webiste => شركة عزل اسطح 49349298 - شركةعزل اسطح بجدة
what file should i update from astrowind to just make it appear like the name_of_article

**Comment by Harold-Anderson** [comment]
From your website, it looks like you figured it out.  Congratulations! @abdelkareemkobo
